tests/codegen/builtin_ctx_field.cpp

```diff
@@ -4,7 +4,7 @@ namespace bpftrace::test::codegen {
 
 TEST(codegen, builtin_ctx_field)
 {
-  std::string prog = R"END(
```

Comment:
this didn't need to change but don't think it makes a difference

Reply:
I've been removing these cases and just standing on the multi-line literal strings, since many are different. Can you just make this `R"(`?

Reply:
Done.